scripts/find_and_tag_cdc_stable_candidate.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Convenience script for finding and tagging a custom DC service image that
+# corresponds to the latest website prod release. Note that this will change
+# which image the stable-candidate tag applies to.
+
+# Usage: ./scripts/find_and_tag_cdc_stable_candidate.sh
+
+set -e
+
+###### BEGIN CONFIG ######
+
+# Tag of the website commit that we'll attempt to find a matching CDC image for.
+WEBSITE_PROD_TAG="prod"
+
+# Tag to be added to the CDC image that is found.
+STABLE_CANDIDATE_TAG="stable-candidate"
```

Comment:
We eventually want to tag as "stable". Is "stable-candidate" temporary until we have tagging working fully?

Reply:
The way I envisioned the workflow was (1) automatically tag stable-candidate (2) verify tests pass against it (3) manually change the tag from stable-candidate to stable. Thoughts?

Reply:
I'm thinking that since autopush sanity would've already kicked in, we'd know before hand if the version had any issues. WDYT?

Reply:
It would have kicked in, but it might not have succeeded. I think it'd be good to have a manual verification of test results before finalizing that the image is stable.

Reply:
SG. 

A couple questions:
* Are you thinking we'd have a cloud run instance pointing to stable-candidate and use that for testing?
* We need to tag the git commit with the `customdc_stable` tag as well. Would we be tagging it with `customdc_stable_candidate` first and update to `customdc_stable` after the manual verification?